### ISSUE TITLE
Remove SENDCOM flags

### ISSUE THREAD

**Issue by WalterKolczynski-NOAA** [NOAA-EMC/global-workflow#1501]
### What new functionality do you need?
Workflow is inconsistent in its use of `SENDCOM` to gate movement of files to COM. We've gotten confirmation from NCO that these gates are no longer needed.

### What are the requirements for the new functionality?
All of the copies to COM that are currently gated by `$SENDCOM` should be changed to just always happen.

### Acceptance Criteria
No more instances of `$SENDCOM` in the workflow

**Comment by HenryRWinterbottom** [comment]
@WalterKolczynski-NOAA I can work on this, but how do you suggest we test this based on your comment "This is going to be a pain to test, since jobs rely on data from previous jobs to be in COM in order to work." above?

Do you have someone in mind who we could ask about testing the fixes? If so, I will begin working on this and ask them to run the appropriate experiments such that we can evaluate the necessary changes.

**Comment by WalterKolczynski-NOAA** [comment]
I think we can hold off on this one. @aerorahul was questioning whether we even need to support it. If we do go through with it, testing will just be a slow process of running a job with it off, making sure the output is created in `$DATA` but not COM, then rerunning the job with the switch on to put it in the right place in COM before moving on to the next one. If @TerrenceMcGuinness-NOAA can get job-by-job testing in order, that might help.

**Comment by HenryRWinterbottom** [comment]
@WalterKolczynski-NOAA OK, sounds good. Thank you for clarifying. Is this issue remains as such, please assign to me and I can work/learn how to resolve this.

**Comment by aerorahul** [comment]
@WalterKolczynski-NOAA 
`SENDCOM` as described by EE2 is _Boolean variable to control data copies to $COMOUT_.  There is no EE2 requirement that I can tell that says, all data that is created in `DATA` and needs to be copied to `COMOUT` should/must to be wrapped in this if-block controlled by `SENDCOM`.

Based on that, I would remove every instance of `SENDCOM` from the scripts as there isn't enough information on when and how should this variable be used.

If it comes to that EE2 is updated that enforces this requirement, we can introduce `SENDCOM`.

Thoughts?

**Comment by WalterKolczynski-NOAA** [comment]
@HenryWinterbottom-NOAA We got word from NCO that `SENDCOM` is no longer required, so we've rewritten this issue to be about removing the existing `SENDCOM` gates instead of adding the missing ones. You are welcome to proceed on this if you still want to take it on.

**Comment by HenryRWinterbottom** [comment]
@WalterKolczynski-NOAA Thanks for following up.

Yes, I'll pick it up.
